design/ceph/ceph-managed-disruptionbudgets.md

```diff
@@ -52,6 +52,8 @@ a common way of deploying kubernetes. This will also work to mitigate manual dra
 
 When an node is drained, we will also delay it's DOWN/OUT process by placing a noout on that node. We will remove that noout after a timeout.
 
+An OSD can be down due to reasons other than node drain, say, disk failure. In such a situation, if the pgs are unhealthy then rook will create blocking PodDisruptionBudget on other failure domains to prevent further node drains on them. `noout` flag won't be set on node this is case. If the OSD is down but all the pgs are active+clean, the cluster will be treated as fully healhty. The default PodDisruptionBudget (with maxUnavailable=1) will be added back and the blocking ones will be deleted.
```

Comment:
```suggestion
An OSD can be down due to reasons other than node drain, say, disk failure. In such a situation, if the pgs are unhealthy then rook will create a blocking PodDisruptionBudget on other failure domains to prevent further node drains on them. `noout` flag won't be set on node this is case. If the OSD is down but all the pgs are `active+clean`, the cluster will be treated as fully healthy. The default PodDisruptionBudget (with maxUnavailable=1) will be added back and the blocking ones will be deleted.
```

Reply:
done.